civictechprojects/static/css/partials/_LogInController.scss

```diff
@@ -88,3 +88,20 @@
   height: 55px;
   cursor: pointer;
 }
+
+.LogInController-google svg{
+  width: 40px;
+  height: 50px;
+}
+
+.LogInController-facebook svg{
+
+}
+
+.LogInController-github svg{
+
+}
+
+.LogInController-linkedin svg{
+
+}
```

Comment:
Code cleanliness, suggest removing all of this. If we need to specify a rule for fb/git/linkedin we can do so.